routing-across-the-meshnet.md

```diff
@@ -33,23 +33,9 @@ Say, Alice and Bob are friends. Alice is already on the mesh network, and Bob wa
 * Alice and Bob both export their own identity json files.
 * Alice and Bob exchange their identity json files. Bob has Alice's identification file, and vice-a-versa.
 * Both Alice and Bob import their friend's identity.
-* Either Bob OR Alice may connect to eachother -- at this point it doesnt't matter who connects to who, as they are both authenticated with one another. 
+* Either Bob OR Alice may connect to eachother -- at this point it doesnt't matter who connects to who, as they are both authenticated with one another.
 * Once a connection is made, Alice will send Bob a list of everyone she can communicate with (node information about the whole network).
-* Bob can now send messages to anyone on the network. 
-
-
-### P2P Messaging / General Encryption
```

Comment:
the reason we don't need to specify the details of this, is because NaCl has only one way to do public key encryption